components/compass-runtime-agent/README.md

```diff
@@ -57,10 +56,10 @@ To rename the Secret containing the CA cert, you must specify these environment
 - **APP_CA_CERT_SECRET_TO_MIGRATE**
 - **APP_CA_CERT_SECRET_KEYS_TO_MIGRATE**
 
-## Generating Custom Resource client
+## Generating custom resource client
```

Comment:
Why lowercase?

Reply:
Because that's what we have in our [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#terminology).